www/web-proxy-useracl/src/opnsense/mvc/app/views/OPNsense/ProxyUserACL/index.volt

```diff
@@ -26,17 +26,99 @@ POSSIBILITY OF SUCH DAMAGE.
+<script type="text/javascript" src="/ui/js/OPNsense/ProxyUserACL/bootstrap-datetimepicker.min.js"></script>
 <script>
-
     $(document).ready(function () {
-        grid = $("#grid-acl").UIBootgrid(
+        $("#grid-users").UIBootgrid(
+            {
+                'search': '/api/proxyuseracl/users/searchUser',
+                'get': '/api/proxyuseracl/users/getUser/',
+                'set': '/api/proxyuseracl/users/setUser/',
+                'add': '/api/proxyuseracl/users/addUser/',
+                'del': '/api/proxyuseracl/users/delUser/',
+            }
+        );
+
+        $("#grid-arps").UIBootgrid(
+            {
+                'search': '/api/proxyuseracl/arps/searchArp',
+                'get': '/api/proxyuseracl/arps/getArp/',
+                'set': '/api/proxyuseracl/arps/setArp/',
+                'add': '/api/proxyuseracl/arps/addArp/',
+                'del': '/api/proxyuseracl/arps/delArp/',
+            }
+        );
+
+        $("#grid-srcs").UIBootgrid(
+            {
+                'search': '/api/proxyuseracl/srcs/searchSrc',
+                'get': '/api/proxyuseracl/srcs/getSrc/',
+                'set': '/api/proxyuseracl/srcs/setSrc/',
+                'add': '/api/proxyuseracl/srcs/addSrc/',
+                'del': '/api/proxyuseracl/srcs/delSrc/',
+            }
+        );
+
+        $("#grid-dsts").UIBootgrid(
+            {
+                'search': '/api/proxyuseracl/dsts/searchDst',
+                'get': '/api/proxyuseracl/dsts/getDst/',
+                'set': '/api/proxyuseracl/dsts/setDst/',
+                'add': '/api/proxyuseracl/dsts/addDst/',
+                'del': '/api/proxyuseracl/dsts/delDst/',
+            }
+        );
+
+        $("#grid-domains").UIBootgrid(
+            {
+                'search': '/api/proxyuseracl/domains/searchDomain',
+                'get': '/api/proxyuseracl/domains/getDomain/',
+                'set': '/api/proxyuseracl/domains/setDomain/',
+                'add': '/api/proxyuseracl/domains/addDomain/',
+                'del': '/api/proxyuseracl/domains/delDomain/',
+            }
+        );
+
+        $("#grid-agents").UIBootgrid(
+            {
+                'search': '/api/proxyuseracl/agents/searchAgent',
+                'get': '/api/proxyuseracl/agents/getAgent/',
+                'set': '/api/proxyuseracl/agents/setAgent/',
+                'add': '/api/proxyuseracl/agents/addAgent/',
+                'del': '/api/proxyuseracl/agents/delAgent/',
+            }
+        );
+
+        $("#grid-mimes").UIBootgrid(
+            {
+                'search': '/api/proxyuseracl/mimes/searchMime',
+                'get': '/api/proxyuseracl/mimes/getMime/',
+                'set': '/api/proxyuseracl/mimes/setMime/',
+                'add': '/api/proxyuseracl/mimes/addMime/',
+                'del': '/api/proxyuseracl/mimes/delMime/',
+            }
+        );
```

Comment:
since they all look almost the same, you may want to generate them in a loop instead of copying. I do this in the nginx plugin for example. 
For the last one, the key value pair would look like:

```json
{
"mimes": "Mime"
}
```

The key could be used for the controller name as well as the selector and the value for the action.
This is just an idea.